package.json

```diff
@@ -9,7 +9,7 @@
     "lint": "node ./node_modules/eslint/bin/eslint **/*.ts",
     "lint-fix": "node ./node_modules/eslint/bin/eslint --fix **/*.ts",
     "build": "npm install && npm run update_api && npm run protoc && tsc",
-    "test": "node ./node_modules/jest/bin/jest.js --runInBand",
+    "test": "jest",
```

Comment:
--runInBand - I added it to run it in same process and sequentially. is it turned on by default?

Reply:
Didn't know it was intentional. I can add it back in next PR.

Reply:
While this PR is still open, just add it back

Reply:
Updated